server/src/modules/boards/boards.service.ts

```diff
@@ -19,9 +21,12 @@ export class BoardsService {
     private readonly res: ResponseService,
   ) {}
 
-  async getBoardById(boardId: string): Promise<BoardResponseObject> {
+  async getBoardById(
+    boardId: string,
+    userId: string,
+  ): Promise<BoardResponseObject> {
     const board = await this.findBoardById(boardId);
-    return this.res.toResponseBoard(board);
+    return this.res.toResponseBoard(board, userId);
```

Comment:
Please, dont modify `toResponseBoard`,
You can add your custom method for handling boards with additional fields, for example `toResponseUserRelatedBoard` or sth like that

It's better to follow backward compatibility rule